File: POOEncapsulamento/app/src/main/java/br/com/matheus/encapsulamento/book.java
package br.com.matheus.encapsulamento;

public class book {
    private String title;
    private String author;
    private String editor;
    private int pages;
    private String location;
    private int yearPub;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getEditor() {
        return editor;
    }

    public void setEditor(String editor) {
        this.editor = editor;
    }

    public int getPages() {
        return pages;
    }

    public void setPages(int pages) {
        this.pages = pages;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public int getYearPub() {
        return yearPub;
    }

    public void setYearPub(int yearPub) {
        this.yearPub = yearPub;
    }
}

File: POOEncapsulamentoKotlin/app/src/main/java/br/com/matheus/pooencapsulamentokotlin/Book.kt
package br.com.matheus.pooencapsulamentokotlin

class Book (

    private var _title: String,
    private var _author: String,
    private var _pages: Int,
    private var _yearPub: Int,
    private var _editor: String,
    private var _location: String

){
    var title: String
        get() = _title
        set(valor){
            _title = valor
        }
    var author: String
        get() = _author
        set(valor){
            _author = valor
        }
    var pages: Int
        get() = _pages
        set(valor){
            _pages = valor
        }
    var yearPub: Int
        get() = _yearPub
        set(valor){
            _yearPub = valor
        }
    var editor: String
        get() = _editor
        set(valor){
            _editor = valor
        }

    var location: String
        get() = _location
        set(valor){
            _location = valor
        }
}
